src/main/java/org/apache/commons/lang3/builder/DiffResult.java

```diff
@@ -88,6 +88,24 @@
         }
     }
 
+    /**
+     * <p>Gets the object the right object has been compared to.</p>
+     * @since 3.10
+     * @return the left object of the diff
+     */
+    public Object getLeft() {
+        return this.lhs;
+    }
+
+    /**
+     * <p>Gets the object the left object has been compared to.</p>
+     * @ 3.10
```

Comment:
Nope :-( Use `@since` not `@`. Also, `@since` should be the _last_ tag.

Reply:
damn' sorry for the typo, ok for putting it in last
also got back to "Returns" for consistency with other Javadocs comments